dot/sync/syncer.go

```diff
@@ -424,6 +439,27 @@ func (s *Service) handleRuntimeChanges(newState *rtstorage.TrieState) error {
 		return ErrEmptyRuntimeCode
 	}
 
+	codeSubBlockHash, err := s.codeSubstitutedState.LoadCodeSubstitutedBlockHash()
+	if err != nil {
+		logger.Debug("problem checking code substituted block hash", "error", err)
```

Comment:
return err

Reply:
I updated codeSubstitutedState.LoadCodeSubstitutedBlockHash does not return error.